CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 - Fixed the SIGPIPE signal handler so Firecracker no longer exits. The signal
   is still recorded in metrics and logs.
+- Fixed ballooning API definitions to include "MiB" instead of "MB".
```

Comment:
I'd add a note to say that this was only a syntactic change. Semantically, the passed value was still treated as MiB.

Reply:
Is saying "fixed ballooning _API definitions_" confusing? I think in the case where we changed some device functionality (such as using MiB when we were previously using MB) would be phrased completely different, like "changed units used by balloon device from MB to MiB".

Reply:
For a customer that maybe was not aware of the issue, it could be confusing. Anyway, it's a nit

Reply:
nit: support what Alin is saying, could be explicit that it's just a rename of a field and nothing else

Reply:
Done.